### PR TITLE
[SHIP-6153] Add all orgnization related endpoint documentation

### PR DESCRIPTION
**Pull request by Hammo1337**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -436,6 +436,371 @@ Definition of the object attributes
...
+
+### POST
+Using the following endpoint you can create a new organization, you will need to provide a JSON body with all of the required details. Organizations are unique to their `externalIds`.
+
+https://public.shipamax-api.com/api/v2/Organizations
+
+>Example JSON body to create an organization
+```json
+{
+    "externalId": "A",
```
Comment (wibblymat) [inline source/index.html.md:450]: This should match with what is returned in the next example
```suggestion
    "externalId": "FF",
```

File: source/index.html.md [hunk 2]
```diff
@@ -436,6 +436,371 @@ Definition of the object attributes
...
+    "carrier": false,
+    "consignee": true,
+    "creditor": true,
+    "forwarder": false,
+    "debtor": false,
+    "shipper": false,
+    "active": false,
+    "updated": "2020-01-01"
```
Comment (wibblymat) [inline source/index.html.md:458]: Shouldn't ever set the `updated` column, we auto fill this in for them.
```suggestion
```

File: source/index.html.md [hunk 3]
```diff
@@ -436,6 +436,371 @@ Definition of the object attributes
...
+  "externalId": "FF",
+  "carrier": false,
+  "consignee": true,
+  "creditor": true,
+  "forwarder": false,
+  "debtor": false,
+  "shipper": false,
+  "active": false,
+  "updated": "2020-01-01T00:00:00.000Z",
+  "companyId": 100000
```
Comment (wibblymat) [inline source/index.html.md:475]: I think we can use the "hidden" property of the LB3 JSON schema to make this not return if you want.

File: source/index.html.md [hunk 4]
```diff
@@ -436,6 +436,371 @@ Definition of the object attributes
...
 ```
 
+## Organization Endpoint
+You can upload organization data to Shipamax. Organization data can be retrieved by ID, or by a query filter. You can also update an organization's details and delete an organization by ID.
+
+### POST
+Using the following endpoint you can create a new organization, you will need to provide a JSON body with all of the required details. Organizations are unique to their `externalIds`.
+
+https://public.shipamax-api.com/api/v2/Organizations
+
```
Comment (wibblymat) [inline source/index.html.md:446]: This doesn't match the style of the other documentation in the same file. No need to include the domain name. The style of the API endpoint immediately before this one, for example, is:
| Endpoint                         | Verb  | Description                                                                       |
| -------------------------------- | ----- | --------------------------------------------------------------------------------- |
| /FileGroups/{id}/validationResult | POST  | Submit a new validationResult making it the lastValidationesult of the FileGroup  | 


The description of the endpoint should also focus on what it _means_, if that makes sense.

```suggestion
## Organization Endpoint
An Organization represents a business or other third-party that is referenced from the documents that Shipamax processes, like the shipper or consignee for a bill of lading. Each Organization must have a unique identifier provided by you, usually the identifier used in your own system, which we call `externalId` and is returned in details of processed documents. However, the `id` in the Organization API calls is the Shipamax specific ID.

| Endpoint                         | Verb  | Description                                                                       |
| -------------------------------- | ----- | --------------------------------------------------------------------------------- |
| /Organizations | GET | Retrieve a list of your Organizations that match a filter  | 
| /Organizations | POST  | Create a new Organization  | 
| /Organizations/{id} | PATCH  | Update an Organization by specifying the fields that have changed  | 
| /Organizations/{id} | DELETE | Remove an Organization from the system  | 
```
There are probably other inconsistencies too.

**Review by wibblymat — APPROVED** [review]

File: source/index.html.md [hunk 5]
```diff
@@ -436,6 +436,421 @@ Definition of the object attributes
...
+}
+```
+
+### GET (Using Query Filter)
+
+Retrieve Organizations via `GET` request to `https://public.shipamax-api.com/api/v2/Organizations` and sending in a JSON body to return all Organizations matching the given filter body, or none for all Organizations.
+
+> The GET Organizations query filter body requires JSON structured like this:
+```json
+{
```
Comment (laddiko) [inline source/index.html.md:538]: Can we please add an example of a using the GET with filter

Comment (Hammo1337) [reply source/index.html.md:538]: This part is the example itself?

**Pull request merged**